addons/armaos/functions/fnc_terminal_updateOutput.sqf

```diff
@@ -26,8 +26,10 @@ private _output = [];
 	_output pushBack _buffer;
 } forEach _terminalBufferVisible;
 
-_outputControl ctrlSetStructuredText (composeText _output);
-ctrlSetFocus _outputControl;
+_consoleOutput ctrlSetStructuredText (composeText _output);
+
+private _consoleInput = _consoleOutput getVariable ["AE3_consoleInput", controlNull];
+ctrlSetFocus _consoleInput;
```

Comment:
Why do we need to set the focus again?
If this is important, please at a comment

Reply:
@GermanHydrogen  The focus changes in cases where the player clicks on other UI elements like the buttons in the header. I don't think that it's strictly necessary but this way we ensure that the focus is on the input field.